addons/pos_self_order/tests/test_self_order_kiosk.py

```diff
@@ -156,3 +157,16 @@ def test_self_order_kiosk_combo_sides(self):
         self.pos_config.current_session_id.set_opening_control(0, "")
         self_route = self.pos_config._get_self_order_route()
         self.start_tour(self_route, "test_self_order_kiosk_combo_sides")
+
+    def test_self_order_kiosk_combo_qty_max_free(self):
+        setup_product_combo_items(self)
+        self.desks_combo.write({'combo_item_ids': [Command.unlink(item.id) for item in self.desks_combo.combo_item_ids[1:]], 'qty_max': 3})
+        self.office_combo.write({'combo_ids': [Command.unlink(item.id) for item in self.office_combo.combo_ids if item.id != self.desks_combo.id]})
```

Comment:
```suggestion
        combo = self.env["product.combo"].create({
            "name": "Desk Accessories Combo",
            "combo_item_ids": [
                Command.create({
                    "product_id": self.fanta.id,
                    "qty_max": 3,
                }),
            ],
        })
```
make it simple and clear.
If you do that, also change the product in the tour that you select

Reply:
Hello @adgu-odoo 
The goal is to use the existing product combo and its main combo product without creating a new one.
If we create a new product.combo, we are also required to create a new product.product for the combo, which we want to avoid.